src/sezzle-checkout-installment-widget/sezzle-checkout-installment-widget.js

```diff
@@ -339,12 +354,10 @@ function renderInstallmentWidget(checkoutTotal, serviceRegion, currencySymbol){
 			width: 100%;
 			margin: 40px 0px -35px 0px;
 		}
-		.sezzle-modal-payment-percent, .sezzle-modal-payment-schedule {
+		.sezzle-modal-payment-percent{
```

Comment:
If you haven't linted already, be sure to do that after making the other changes listed in this review.